src/main/java/com/useful/ucars/ucars.java

```diff
@@ -688,16 +688,25 @@ public final Boolean isBlockEqualToConfigIds(List<String> rawIds, Block block) {
 					return true;
 				}
 			} else {
-				final String mat = parts[0];
-				final int data = Integer.parseInt(parts[1]);
-				final int bdata = block.getData(); //TODO Alternative to .getData()
-				if (mat.equalsIgnoreCase(block.getType().name()) && bdata == data) {
+				//TODO Alternative to .getData()
+				if (ItemStackFromId.equals(raw,block.getType().name().toUpperCase(),block.getData())) {
 					return true;
 				}
 			}
 		}
 		return false;
 	}
+	public final Boolean isBlockEqualToConfigIds(List<String> rawIds, String blockName) {
```

Comment:
Ought to support or convert existing config values. Perhaps prefer to convert to standard bukkit itemstack config representation